docs/guide/built-in-functions.md

```diff
@@ -224,19 +224,19 @@ Total number of functions: **{{ $page.functionsCount }}**
 
 ### Lookup and reference
 
-| Function ID | Description | Syntax |
-| :--- | :--- | :--- |
-| CHOOSE | Uses an index to return a value from a list of up to 30 values.| CHOOSE(Index; Value1; ...; Value30) |
-| COLUMN | Returns column number of a given reference or formula reference if argument not provided. | COLUMNS([Reference]) |
-| COLUMNS | Returns the number of columns in the given reference. | COLUMNS(Array) |
-| FORMULATEXT | Returns a formula in a given cell as a string. | FORMULATEXT(Reference) |
-| HLOOKUP | Searches horizontally with reference to adjacent cells to the bottom. | HLOOKUP(Search_Criterion; Array; Index; Sort_Order) |
-| INDEX | Returns the content of a cell, specified by row and column number, or an optional range name. | INDEX(Reference; Row; Column; Range) |
-| MATCH | Returns the relative position of an item in an array that matches a specified value. | MATCH(Searchcriterion; Lookuparray; Type) |
-| OFFSET | Returns the value of a cell offset by a certain number of rows and columns from a given reference point. | OFFSET(Reference; Rows; Columns; Height; Width) |
-| ROW | Returns row number of a given reference or formula reference if argument not provided. | ROW([Reference]) |
-| ROWS | Returns the number of rows in the given reference. | ROWS(Array) |
-| VLOOKUP | Searches vertically with reference to adjacent cells to the right. | VLOOKUP(Search_Criterion; Array; Index; Sort_Order) |
+| Function ID | Description                                                                                                            | Syntax                                              |
+| :--- |:-----------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|
+| CHOOSE | Uses an index to return a value from a list of up to 30 values.                                                        | CHOOSE(Index; Value1; ...; Value30)                 |
+| COLUMN | Returns column number of a given reference or formula reference if argument not provided.                              | COLUMNS([Reference])                                |
+| COLUMNS | Returns the number of columns in the given reference.                                                                  | COLUMNS(Array)                                      |
+| FORMULATEXT | Returns a formula in a given cell as a string.                                                                         | FORMULATEXT(Reference)                              |
+| HLOOKUP | Searches horizontally with reference to adjacent cells to the bottom.                                                  | HLOOKUP(Search_Criterion; Array; Index; Sort_Order) |
+| INDEX | Returns the content of a cell, specified by row and column number. The column parameter is optional and defaults to 1. | INDEX(Range; Row[; Column])                         |
```

Comment:
```suggestion
| INDEX | Returns the contents of a cell specified by row and column number. The column number is optional and defaults to 1. | INDEX(Range; Row [; Column])                         |
```

- It's content**s**, not content
- The comma shouldn't be there (changes the meaning)
- IMO either "the `Column` parameter" or "the column number"

Reply:
Thanks a lot!